# HG changeset patch
# User gbtju85@gmail.com
#

Replace bios_relocate hook with bios_load hook in hvmloader.

Sign-off-by: Bei Guan <gbtju85@gmail.com>

diff -r 42edf1481c57 tools/firmware/hvmloader/seabios.c
--- a/tools/firmware/hvmloader/seabios.c	Fri Jul 22 08:55:19 2011 +0100
+++ b/tools/firmware/hvmloader/seabios.c	Sun Jul 24 02:22:42 2011 +0800
@@ -132,6 +132,7 @@
     .image_size = sizeof(seabios),
 
     .bios_address = SEABIOS_PHYSICAL_ADDRESS,
+    .load = NULL,
 
     .load_roms = 0,
 
@@ -141,8 +142,6 @@
     .bios_info_setup = seabios_setup_bios_info,
     .bios_info_finish = seabios_finish_bios_info,
 
-    .bios_relocate = NULL,
-
     .vm86_setup = NULL,
     .e820_setup = seabios_setup_e820,
 
